intro-to-dev-environment/intro-to-command-line.md

```diff
@@ -0,0 +1,178 @@
+# Intro to Command Line
+
+## Learning Goals
+
+- Create commands to navigate the file system through the command line using cd, ls, and pwd.
+- Apply commands to create files and directories in command line using touch and mkdir.
+- Apply the python3 command in the command line to run a Python script.
+
+## Introduction
+
+As a programmer, the terminal is one of our most important tools, and we will run thousands of commands in the terminal over our career. Before we get there, we should spend time familiarizing ourselves with the command line.
+
+## Vocabulary and Synonyms
+
+| Vocab          | Definition                                                                                                                                                     | Synonyms  | How to Use in a Sentence                                                                                                                                                                                                         |
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| interface      | A medium that takes information from one source, and passes it to another source.The interface usually makes a meaningful translation between the two sources. | -         | "The command line and the terminal is one interface for how we interact with the file system and run programs on our computer," "That program has a graphical user interface, and there are buttons and images to interact with" |
```

Comment:
```suggestion
| interface      | A medium that takes information from one source, and passes it to another source. The interface usually makes a meaningful translation between the two sources. | -         | "The terminal is one interface for how we interact with the file system and run programs on our computer", "That program has a graphical user interface, and there are buttons and images to interact with" |
```

Reply:
The space is a good catch!

In terms of English grammar, I feel fairly confident that the commas should go inside of the quotation marks ([grammarly rules](https://www.grammarly.com/blog/quotation-marks) under the header "quotation marks and other punctuation marks")

I could see us putting the comma on the outside of the quote if we're thinking that the quote is a string literal ;) lolol, then it might make sense to me if the comma is outside. Or do you have another reason?

Reply:
LOL Yes! I was thinking of this column containing 'lists' of sentences.